__tests__/app.test.js

```diff
@@ -107,6 +86,55 @@ describe("GET /api/users", () => {
       })
   })
   test("throws an error if route is wrong", () => {
-      return request(app).get("/api/banana").expect(404);
-    });
-  });
\ No newline at end of file
+      return request(app).get("/api/banana").expect(404)
+    })
+})
+
+describe.only("PATCH /api/articles/:article_id", () => {
```

Comment:
Remove .only and then re-run your tests please! All tests must be confirmed to be passing before moving on to further functionality.

Reply:
reruned an all test passed